Engine/Module/ModuleRender.h

```diff
@@ -37,6 +40,12 @@ class ModuleRender : public Module
 	ComponentMesh* CreateComponentMesh();
 	void RemoveComponentMesh(ComponentMesh* mesh_to_remove);
 	void GenerateQuadTree();
+	void InsertAABBTree(GameObject* game_object);
+	void RemoveAABBTree(GameObject* game_object);
+	void UpdateAABBTree(GameObject* game_object);
+	void DeleteAABBTree();
+	void CreateAABBTree();
+	void DrawAABBTree() const;
```

Comment:
Most of these functions should be inside of `AABBTree` class, or ideally inside a new module that manages SpacePartitioning

Reply:
These functions exist because OLAABBTree is private as the quadtree is, I could made OLAABBTree public and then erase these functions